params/chainspecs/hoodi.json

```diff
@@ -0,0 +1,37 @@
+{
+  "chainName": "hoodi",
+  "chainId": 560048,
+  "consensus": "ethash",
+  "homesteadBlock": 0,
+  "daoForkBlock": 0,
```

Comment:
It's probably immaterial, but we don't need the following blocks here: `daoForkBlock`, `muirGlacierBlock`, `arrowGlacierBlock`, `grayGlacierBlock`. `daoForkBlock` is about an irregular state transition peculiar to the mainnet, while the glacier blocks are about postponing the time difficulty bomb (not applicable to Hoodi).